### PR TITLE
Update onboarding doc

### PR DESCRIPTION
**Pull request by PatrickXYS**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
Update onboarding doc to not configure webhook anymore, since we have configured organization-level webhook.

**Checklist:**
If PR related to **Optional-Test-Infra**,

- [ ] Changes have been generated to `aws/GitOps` folder: 
    1. `cd aws`
    2. `make optional-generate`

### CODE REVIEW

**Comment by PatrickXYS** [comment]
@theofpa Can you help approve?

File: aws/Doc/onboarding.md [hunk 1]
```diff
@@ -4,13 +4,12 @@ This doc describes how to onboard new kubeflow repository using optional-test-in
...
 1. Grant repo access to aws-kf-ci-bot, check [PR](https://github.com/kubeflow/internal-acls/pull/373)
-2. Add Webhook to the repo, ping YaoXiao@ for help [TODO(PatrickXYS): Need to be automated]
-3. Configure Presubmit on Optional-Test-Infra Prow Cluster, check [config.yaml](https://github.com/kubeflow/testing/blob/master/aws/User/clusters/kubeflow-shared-test-infra-poc/namespaces/prow/configmap/config.yaml#L119-L131)
+2. Configure Presubmit on Optional-Test-Infra Prow Cluster, check [config.yaml](https://github.com/kubeflow/testing/blob/master/aws/User/clusters/kubeflow-shared-test-infra-poc/namespaces/prow/configmap/config.yaml#L119-L131)
 
 ## Instructions 
 A geneic Argo Workflow / Tekton Pipelinerun diagram attached [here](https://github.com/kubeflow/testing/tree/master/aws/Picture/GenericArgoWorkflow.png)
 
-Below is the steps for WG who want to write a simple E2E test cases
+Below is the steps for WG who want to write a generic E2E test cases
```
Comment (theofpa) [inline aws/Doc/onboarding.md:12]:
```suggestion
Below are the steps for WG who want to write a generic E2E test cases
```

**Review by theofpa — CHANGES_REQUESTED** [review]
thanks @PatrickXYS

File: aws/Doc/onboarding.md (hunk 1 above)
Comment (PatrickXYS) [reply aws/Doc/onboarding.md]: Thanks, addressed in the new commit

**Pull request merged**